diff --git a/lacord-scm-4.rockspec b/lacord-scm-4.rockspec
index 3b27de7..3ea036a 100644
--- a/lacord-scm-4.rockspec
+++ b/lacord-scm-4.rockspec
@@ -32,11 +32,6 @@ build = {
         ["lacord.util.archp"] = "src/archp.c",
         ["lacord.cli"] = "lua/lacord/util/cli_default.lua",
         ["acord"] = "lua/lacord/util/cli_auto.lua",
-        ["lacord.ext.shs"] = "ext/shs/shs.lua",
         ["lacord.outgoing-webhook-server"] = "lua/lacord/wrapper/outgoing-webhook-server.lua",
-
-        ["internationalize"] = "ext/internationalize/internationalize/init.lua",
-        ["internationalize.interpolation"] = "ext/internationalize/internationalize/interpolation.lua",
-        ["internationalize.plural"] = "ext/internationalize/internationalize/plural.lua"
     }
 }
\ No newline at end of file
